### DISCUSSION TITLE
How to get the following behaviour in NvimTree...

### DISCUSSION THREAD

**Discussion by colliderman** [nvim-tree/nvim-tree.lua#1686] [General]
Hi. When selecting a directory in NvimTree I'd like it to only show the directories and files inside of the selected directory, without having to change my working directory to the selection. In other words, the list would get filtered down to only what's in the directory I have selected with the Enter key. This is the way Netrw behaves, for example.

As it stands, selecting (pressing Enter) on a directory unfolds it but still shows everything else. I'm sure this must be a simple configuration setting but I can't for the life of me figure out what it is! I've tried 'sync_root_with_cwd' and 'hide_root_folder = true' but these aren't it. Any help is appreciated!

**Comment by alex-courtis** [comment]
The `cd` command (default `<C-]>` and`<2-RightMouse>`) may achieve your use case.

`dir_up` (default `-`) is the complement.

See `:help nvim-tree-default-mappings`
